cvmfs/libcvmfs.cc

```diff
@@ -22,11 +22,14 @@
 #include "libcvmfs_int.h"
 #include "logging.h"
 #include "smalloc.h"
+#include "statistics.h"
 #include "util.h"
 
 using namespace std;  // NOLINT
 
 
+perf::Statistics statistics_;
```

Comment:
That was the source of the previous problem, where by mistake I was using different statistics objects. After creating that global everything inside libcvmfs uses that statistics object and there are no clashes.
However now there is no problem with the ShortString class, so maybe we could think about playing with different statistics objects, but I have the sensation it would be playing with fire.